lib/commands/gesture.js

```diff
@@ -24,21 +24,104 @@ commands.click = async function (el) {
   }
 };
 
+helpers.isSameGestures = function (gestures, candidates) {
```

Comment:
Since this function does not use anything on the driver object, I would rather see it (and the `gesturesChainToString` function) as a standalone function, which is then exported from this file for testing purposes.